src/main/java/com/github/chhsiao90/nitmproxy/TlsContext.java

```diff
@@ -7,6 +7,8 @@
 import io.netty.util.concurrent.Promise;
 
 public class TlsContext {
+
+  private boolean plain;
```

Comment:
Maybe `enabled`? I guessed `TlsContext.isEnabled` is more clear. What do you think?

Reply:
Yes, sure.

Reply:
fixed